ui/src/components/Filter.tsx

```diff
@@ -54,6 +83,17 @@ const Tagline = styled.span`
   padding-bottom: 16px;
 `;
 
+const useStyles = makeStyles(() => ({
+  paper: {
```

Comment:
This was the only way I was able to get the Popper text styled correctly 
![image](https://user-images.githubusercontent.com/19617248/101570970-7594b880-39a5-11eb-9865-a623b933fe46.png)
because it wasn't overriding/injecting the style properly for styled components.
Unfortunately it throws this error when linting  `112:18  error  React Hook "useStyles" is called conditionally. React Hooks must be called in the exact same order in every component render. Did you accidentally call a React Hook after an early return?  react-hooks/rules-of-hooks   `